### DISCUSSION TITLE
Source IP Address(es) for Firewall Rules

### DISCUSSION THREAD

**Discussion by W4JEW** [upptime/upptime#931] [Q&A] [answered]
I checked the documentation and FAQ - I do not see anything that speaks to what IP address(es) the UppTime checks originate from. Some of the origins I want to check have firewall policies that restrict access to the port(s) I want to check, and I need to understand which IP addresses to permit. I realize this may be more of a GitHub question, but I am unsure where to direct the query.

If you happen to know, including the details in your documentation would be great!

I just found one resource that might answer my question... There's a GitHub API endpoint (meta) that provides a list of the IP addresses for various GitHub services. I think the IPs under "actions" would reflect what needs to be permitted for UppTime to work, but I would appreciate it if you could confirm.

https://api.github.com/meta

Thanks!

**Comment by AnandChowdhary** [comment, marked as answer]
Yes, you are exactly right :) Since Upptime uses GitHub Actions, we use the IP address range of Actions.
